src/bin/bb.rs

```diff
@@ -25,6 +39,15 @@ struct BlueBuildArgs {
 
 #[derive(Debug, Subcommand)]
 enum CommandArgs {
+    /// Create a pre-populated GitHub issue with information about your configuration
+    BugReport,
```

Comment:
I would prefer this to be lower in the struct so that the other commands are at the top of the help text.